liver_imaging_analysis/engine/engine.py

```diff
@@ -140,6 +150,36 @@ def get_metrics(self, metrics_name, **kwargs):
             "jaccard" : MeanIoU,
         }
         return metrics[metrics_name](**kwargs)
+    
+    def get_hparams(self, network_name):
+        """
+        used to return a hyperparameters dictionary specific to each network 
+        Parameters
+        ----------
+        network_name: str
+                name of the network to fetch its hyperparameters.
+        Returns
+        -------
+        dict
+            A dictionary containing the hyperparameters for the specified network.
+        """
+        hparams = {
+            "monai_2DUNet" :  {
+            "spatial_dims": config.network_parameters["spatial_dims"],
+            "num_res_units": config.network_parameters["num_res_units"],
+            "bias": config.network_parameters["bias"],
+            "norm": config.network_parameters["norm"],
+            "dropout": config.network_parameters["dropout"],
+            "batch_size": config.training["batch_size"],
+            "optimizer": config.training["optimizer"],
+            "lr_scheduler": config.training["lr_scheduler"],
+            "loss_name": config.training["loss_name"],
+            "metrics": config.training["metrics"],
+            "shuffle": config.training["shuffle"]
+        }
+        }
```

Comment:
would you like to just loop over all keys in the config.network_parameters and config.training? This way it would be more generic than this. Also, you wont need the network_name parameter

Reply:
done, I left 'epochs' key out